ci/testRunner_memcheck_osx.js

```diff
@@ -15,7 +15,7 @@ var testUrls = ["http://pxscene.org/examples/px-reference/gallery/fancy.js",
                 ];
 
 //added different timeout for different pages, as travis server is bit slower and so getting more timeout errors for complex js pages
-var testTimeouts = [5000, 5000, 20000, 420000];
+var testTimeouts = [5000, 5000, 20000, 600000];
```

Comment:
A comment about units would be good here... 600000 Seconds ? or Milliseconds ?

More comments and whitespace help clarity